### ISSUE TITLE
Recent Cinnamon Updates break Extension(s)/Access to Control Centre

### ISSUE THREAD

**Issue by UK82** [void-linux/void-packages#22366]
An update to a number of Cinnamon components last weekend result in the above issue.

Lookingglass log indicates the following as contributing to error:

GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.cinnamon.SettingsDaemon was not provided by any .service files

(Apologies if my reporting etiquette is non conventional, I don't 'do' Github!)

Regards,

Stu

**Comment by ericonr** [comment]
Have you checked if `cinnamon-settings-daemon` is installed?

**Comment by not-chicken** [comment]
Don't know if it's directly related to this issue or not but trying to open cinnamon settings(control-center) gave this output for me:
```
File "/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", line 730, in <module>
    window = MainWindow()
  File "/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", line 309, in __init__
    for module in modules:
  File "/usr/share/cinnamon/cinnamon-settings/modules/cs_info.py", line 10, in <module>
    import distro
ModuleNotFoundError: No module named 'distro'
```
Seems like `python3-distro` should be added as a dependency.

**Comment by UK82** [comment]
Thank you for your mutual responses.. In regard to ericonr's comments, I did check to determine whether cinnamon-settings-daemon was installed and can confirm it is/was..

Re not-chicken's recommendation I confirmed python3-distro wasn't in fact installed and after installing this and rebooting I was then able to once more gain access to the Control Centre. The residual warning notification re a broken extension etc. prevailed but as suspected that was a result of the sole extension I have installed presumably not having caught up with the upstream Cinnamon stuff and once disabled, disappeared on the next reboot.

As a result all sorted, thank you both! :-)
